server/implementation/src/main/java/io/smallrye/graphql/bootstrap/LogPayloadOption.java

```diff
@@ -0,0 +1,21 @@
+package io.smallrye.graphql.bootstrap;
+
+public enum LogPayloadOption {
+
+    OFF("false"),
+    TRUE("true"),
+    FALSE("false"),
+    WITHOUTVARIABLE("withoutvariable"),
+    WITHVARIABLE("withvariable");
+
+    private String logPayload;
+
+    LogPayloadOption(String logPayload) {
+        this.logPayload = logPayload;
+    }
+
+    public void setLogPayload(String logPayload) {
+        this.logPayload = logPayload;
+    }
+
```

Comment:
Add a `String getLogPayload()` method. That will always only return one of `off`, `withoutvariable` or `withvariable`